parsec/backend/postgresql/user_queries/find.py

```diff
@@ -97,45 +105,44 @@ async def query_find_humans(
     omit_revoked: bool,
     omit_non_human: bool,
 ) -> Tuple[List[HumanFindResultItem], int]:
+    if page >= 1:
+        offset = (page - 1) * per_page
+    else:
+        return ([], 0)
+
+    q = _q_human_factory(
+        query=True if query else False,
+        omit_revoked=True if omit_revoked else False,
+        omit_non_human=omit_non_human,
+        offset=offset,
+        limit=per_page,
+    )
     if query:
-
-        if omit_revoked:
-            q = _q_human_factory(query=True, omit_revoked=True, omit_non_human=omit_non_human)
-            args = q(organization_id=organization_id, now=pendulum_now(), query=query)
-
-        else:
-            q = _q_human_factory(query=True, omit_revoked=False, omit_non_human=omit_non_human)
-            args = q(organization_id=organization_id, now=pendulum_now(), query=query)
-
+        args = q(organization_id=organization_id, now=pendulum_now(), query=query)
     else:
-
-        if omit_revoked:
-            q = _q_human_factory(query=False, omit_revoked=True, omit_non_human=omit_non_human)
-            args = q(organization_id=organization_id, now=pendulum_now())
-
-        else:
-            q = _q_human_factory(query=False, omit_revoked=False, omit_non_human=omit_non_human)
-            args = q(organization_id=organization_id, now=pendulum_now())
+        args = q(organization_id=organization_id, now=pendulum_now())
 
     raw_results = await conn.fetch(*args)
-
+    if raw_results:
+        total = raw_results[0][4]
```

Comment:
You can access data through the name which is nicer: `raw_results[0]["full_count"]`

Reply:
Done